src/vmx.rs

```diff
@@ -31,6 +31,72 @@ pub enum MSR {
     Ia32SMBase = 0x0000009e,
 }
 
+// These constants are ugly, but Rust doesn't let you use enums like integers yet.
+// 仕方がない。
+
+// Pin-based controls
+pub const EXTERNAL_INTERRUPT_EXITING: u32 = 1 << 0;
```

Comment:
We should organize this file better. It's kind of a mess right now. We should split out the constants, enums, non-vmx assembly stuff, vmx assembly stuff, and higher level functions each into their own file, probably all under the vmx:: namespace.

Not in this PR though.